macosx/en.lproj/Localizable.strings

```diff
@@ -1126,7 +1126,7 @@
 
 /* Message window -> level
    Message window -> level string */
-"Warn" = "Warn";
+"Warning" = "Warn";
```

Comment:
Do we want to leave it as "Warn" for English? It is advised to make the phrase short, i.e. maybe we want combo widget to be narrow so that UI looks better? Otherwise I'd make it "Warning" as well.

Reply:
...probably better as 'Warning' here too? "Warn" is shorter, but it's a little more awkward in native English